### PR TITLE
Third pull request review

### PR DESCRIPTION
**Pull request by jamestewartjr**

### CODE REVIEW

File: src/binarySearch.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const BinarySearch = (inputArray, searchValue) => {
```
Comment (bairbanu) [inline src/binarySearch.js:1]: Hello! Very clean code, easy to follow step-by-step. One area of improvement I feel could be comments :) Clean code combined with comments will make this super code!

File: src/binarySearch.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        min = inputArray[middle]
+      }
+      else {
+        max = inputArray[middle]
+      }
+    }
+  }
+
+  return -1
+}
```
Comment (ameliavoncat) [inline src/binarySearch.js:20]: I don't have much to add to what @zubairnahmed said - your code is really clean and easy to read, and everything seems to work as intended. 👍 

It might be good to think about edge cases and write a few more tests - but none of the ones I tried off the top of my head broke anything.
